### PR TITLE
chore: updated to use v2 configs

### PR DESCRIPTION
**Pull request by jsalaber**
# Changes

- update to use v2 configs

### CODE REVIEW

File: lib/devcycle-ruby-server-sdk/localbucketing/config_manager.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@ class ConfigManager
     ).void }
     def initialize(sdkKey, local_bucketing, wait_for_init)
       @first_load = true
-      @config_version = "v1"
+      @config_version = "v2"
```
Comment (JamieSinn) [inline lib/devcycle-ruby-server-sdk/localbucketing/config_manager.rb:20]: do we use this anywhere? You'll also need to update the WASM at the same time as this config

Comment (jsalaber) [reply lib/devcycle-ruby-server-sdk/localbucketing/config_manager.rb:20]: yeah it's used to fetch different versions of the config below

File: lib/devcycle-ruby-server-sdk/localbucketing/update_wasm.sh [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-BUCKETING_LIB_VERSION="1.24.2"
+BUCKETING_LIB_VERSION="1.24.3"
```
Comment (JamieSinn) [inline lib/devcycle-ruby-server-sdk/localbucketing/update_wasm.sh:2]: version should be 25.3?

Comment (jsalaber) [reply lib/devcycle-ruby-server-sdk/localbucketing/update_wasm.sh]: yeah forgot to push

**Review by JamieSinn — APPROVED** [review]

**Pull request merged**